hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/helpers/OmPrefixInfo.java

```diff
@@ -184,7 +184,7 @@ public static OmPrefixInfo getFromProtobuf(PersistedPrefixInfo prefixInfo) {
           .getFromProtobuf(prefixInfo.getMetadataList()));
     }
     if (prefixInfo.getAclsList() != null) {
-      opib.setAcls(OzoneAclStorageUtil.fromProtobuf(prefixInfo.getAclsList()));
+      opib.setAcls(OzoneAclUtil.fromProtobuf(prefixInfo.getAclsList()));
```

Comment:
There is a change in Class name hierarcy for OzoneAclInfo from OzoneManagerStorageProtos --> OzoneManagerProtocolProtos
and Om.StorageProtocol.proto used for content saved in rocksdb for encode/decode.
Please check if this have any `compatibility issue`, i.e. before upgrade, having encoded with old way in db, and after upgrade, it should able to retrive same information from db with the change.

Reply:
Thank you for the review. I have tested it, and there was no compatibility issue.

Testing steps:
1. Deploy OM using 1.4.0-rc1 (without this patch)
2. Create a prefix (`ozone sh prefix addacl --acls="group:ozone:w" //dev/vol1/test-bucket/prefix/`)
3. Deploy OM using this patch
4. Read the same prefix (`ozone sh prefix getacl o3://dev/vol1/test-bucket/prefix/`)
5. Add another prefix (`ozone sh prefix addacl --acls="user:testuser:w" //dev/vol1/test-bucket/prefix/`)

Reply:
can we read both prefix and print if comming properly?

Reply:
Yes, both type of prefixes can be read and printed properly.